nomos-cli/src/cmds/disseminate/mod.rs

```diff
@@ -44,15 +44,15 @@ impl Disseminate {
             <NetworkService<Libp2p> as ServiceData>::Settings,
         >(std::fs::File::open(&self.network_config)?)?;
         let (status_updates, rx) = std::sync::mpsc::channel();
-        let bytes: Box<[u8]> = match (&self.data, &self.file) {
-            (Some(data), None) => data.clone().as_bytes().into(),
-            (None, Some(file_path)) => {
-                let file_bytes = std::fs::read(file_path)?;
-                file_bytes.into_boxed_slice()
-            }
-            (Some(_), Some(_)) => return Err("Cannot specify both data and file".into()),
-            (None, None) => return Err("Either data or file must be specified".into()),
+
+        let bytes: Box<[u8]> = if let Some(data) = &self.data {
+            data.clone().as_bytes().into()
+        } else {
+            let file_path = self.file.as_ref().unwrap();
+            let file_bytes = std::fs::read(file_path)?;
+            file_bytes.into_boxed_slice()
         };
+
```

Comment:
nit: why this change?

Reply:
The previous solution, even being readable and concise, the panic message differed from the user friendly warning that Clap produced. The output when user provides the `--author` flag, but forgets the `--message` now is:
```
error: the following required arguments were not provided:
  --message <MESSAGE>

Usage: nomos-cli chat --network-config <NETWORK_CONFIG> --node <NODE> --message <MESSAGE> --author <AUTHOR>
```
If user doesn't provide the author nor message, the app goes into the interactive mode